doc/_doxygen/doxygen-awesome.css

```diff
@@ -150,7 +150,7 @@ html {
     --side-nav-arrow-opacity: 0;
```

Comment:
The changes look ok, you need to tweak the commit message a bit.
Please set the commit message something like
```
doc: Fix typos

Fix typos in the documentation.

Signed-off-by: <add here your email id>
```